### DISCUSSION TITLE
美区的应用商店的MiaoProject

### DISCUSSION THREAD

**Discussion by qingyu308** [Paladinfeng/MiaoProject#136] [Q&A] [answered]
美区的应用商店的MiaoProject可以直接购买嘛 功能是一样的嘛？

**Comment by Paladinfeng** [comment, marked as answer]
- 同样需要按照使用说明激活
- 使用说明 https://github.com/Paladinfeng/MiaoProject/blob/master/HOW-TO-USE.md
- 一定要注意输入的大小写
- 目前只能用现在这个丑丑的图标
- TF组已经满载，现在有测试资格的还是可以正常使用
- 注意：报错「无法更新APP 无法使用您的Apple ID 更新，因为此项目已经退款或通过其他Apple ID购买，或者，登录iTunes Store，请输入Apple ID的密码。」，**请删掉TF版本后重试**
- Telegram: https://t.me/miaoproject

**Reply by Thetigerwu** [reply]
请问MiaoProject还可以购买吗，美区app store搜索MiaoProject 找不到了

**Reply by mingyuerujing** [reply]
同问

**Comment by policeman0077** [comment]
加拿大区搜索不到哟

**Reply by Paladinfeng** [reply]
已全区下架

**Reply by 1345683654** [reply]
以前有的资格现在也到期了，不会再开放资格了吗？

**Reply by qingyu308** [reply]
之前购买的可以正常使用

**Reply by 1345683654** [reply]
我错过了，没在商店里买过，只有TF的测试，应该是今天刚到期了，没法再打开了

**Reply by policeman0077** [reply]
还有啥别的替代吗？

**Reply by 1345683654** [reply]
作者大大能再开放一波测试吗？

**Comment by dgkang** [comment]
跪求作者大大再开放一次正式版购买
